### PR TITLE
Updating master's dashboard.html file

### PR DESCRIPTION
**Pull request by JHuraibi**
New branch to branch off from the repo's current master branch. Then add in content from dashboard.html.

### CODE REVIEW

File: public/dashboard.html [hunk 1]
```diff
@@ -240,8 +252,10 @@ <h2 style="margin-top: 2rem">History</h2>
 	<script>
 		// !! TODO: Docstrings and comments
 		// If Time: Use event listeners instead of onclick (Unobtrusive Javascript)
-		var tableHelper = new FirebaseDAO(dbRef);
 		var tableHelperGrowingCircles = new growingCirclesDAO(dbRef);
```
Comment (JHuraibi) [inline public/dashboard.html:255]: We can change "tableHelperGrowingCircles" name to something along the lines of growingCirclesDAO.

File: public/dashboard.html [hunk 2]
```diff
@@ -259,34 +273,48 @@ <h2 style="margin-top: 2rem">History</h2>
...
+				console.log("Manual Update of Size Was Required.");
+				manualSizeUpdate();
+				setTimeout(temp_drawSymbols, 200);
+			}
+			
+			symbolsDAO.drawSymbols('resultsContainerSymbols', miniCanvasDivSize);
 		}
 
-		function temp_drawGrowingCircles() {
+		function temp_drawFullBars() {
```
Comment (JHuraibi) [inline public/dashboard.html:310]: "temp_" should be removed ASAP.

**Review by fx0285 — APPROVED** [review]
Looks good

**Review by fx0285 — APPROVED** [review]

**Review by JHuraibi — COMMENTED** [review]
Minor edits still required.

**Pull request merged**